_data/toc/php-developer-guide.yml

```diff
@@ -228,7 +228,7 @@ pages:
           url: /extension-dev-guide/searching-with-repositories.html
 
         - label: Configure services as web APIs
-          include_versions: ["2.0", "2.1"]
+          include_versions: ["2.0", "2.1", "2.2"]
```

Comment:
just to confirm it shouldn't be 2.3 as that was added above and 2.2 was removed?

Reply:
This is correct. Since 2.2 no longer needs a "Web API" navigation subsection, I moved this back to the same location as 2.0 and 2.1. Note that lines 195 and 196 define this link for 2.3.

I corrected the missing quotation mark error.